compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirFunctionParameterChecker.kt

```diff
@@ -20,6 +24,25 @@ import org.jetbrains.kotlin.types.AbstractTypeChecker
 object FirFunctionParameterChecker : FirFunctionChecker() {
     override fun check(declaration: FirFunction<*>, context: CheckerContext, reporter: DiagnosticReporter) {
         checkVarargParameters(declaration, context, reporter)
+        checkParameterTypes(declaration, context, reporter)
+    }
+
+    private fun checkParameterTypes(declaration: FirFunction<*>, context: CheckerContext, reporter: DiagnosticReporter) {
+        for (valueParameter in declaration.valueParameters) {
```

Comment:
It looks like we have only one test case for this diagnostic? Since we're going to report this _per_ value parameter, could you double-check FE 1.0 does so while you're on this?

Reply:
OK, I added more untyped parameters to this test case. Also I checked FE1.0 highlights each one of the value parameter with this error.

Reply:
Thanks for adding that. But you need to run FE 1.0 's `DiagnosticTestGenerated` and update `.txt` file as well: https://teamcity.jetbrains.com/buildConfiguration/Kotlin_KotlinForGoogle_AggregateBranch?mode=builds#3369162

Reply:
Aha! Thanks for the help!